### PR TITLE
Fix when there's no user logged in

### PR DESCRIPTION
**Pull request by T0mi**
Fixing issue #49 but not sure if it won't break something else

### CODE REVIEW

File: src/Model/Behavior/FootprintBehavior.php [hunk 1]
```diff
@@ -179,12 +179,14 @@ protected function _injectEntity(EntityInterface $entity, ArrayObject $options,
                 ($when === 'new' && $new) ||
                 ($when === 'existing' && !$new)
             ) {
+                $value = current(Hash::extract(
```
[the remark this answers is not in the document]
Comment (T0mi) [reply src/Model/Behavior/FootprintBehavior.php]: I've tried it and you are right, it returns exactly what we want. Should i change it?

[the remark this answers is not in the document]
Comment (ADmad) [reply src/Model/Behavior/FootprintBehavior.php]: Yes please.

[the remark this answers is not in the document]
Comment (T0mi) [reply src/Model/Behavior/FootprintBehavior.php]: Done

**Comment by ADmad** [comment]
I added a test for the change :)

**Comment by ADmad** [comment]
Right after merging this.

**Comment by T0mi** [comment]
Thanks!

**Comment by ADmad** [comment]
https://github.com/UseMuffin/Footprint/releases/tag/1.2.2

**Comment by Spriz** [comment]
This breaks our unit tests when moving from 1.2.1 to 1.2.2 - anyone else had troubles? Will try and dive into it tomorrow if I have time.

**Pull request merged**